components/templates/src/global_fns.rs

```diff
@@ -8,6 +8,33 @@ use config::Config;
 use utils::site::resolve_internal_link;
 
 
+pub fn make_i18n(config: Config) -> GlobalFn {
+    let languages = config.languages.unwrap();
+    let default_lang = config.default_language.unwrap();
+
+    Box::new(move |args| -> Result<Value> {
+        match args.get("key") {
+            Some(value) => match from_value::<String>(value.clone()) {
+                Ok(key) => {
+                    match args.get("lang") {
```

Comment:
can simplify this by doing `args.get("lang").unwrap_or(default_lang)` (from memory, it might not be `unwrap_or`) to get rid of the match